### PR TITLE
Workbench: Version 2.000;gftools[0.9.33] added

### PR DESCRIPTION
**Pull request by vv-monsalve**
2690e14: [gftools-packager] Workbench: Version 2.000;gftools[0.9.33] added

* Workbench Version 2.000;gftools[0.9.33] taken from the upstream repo https://github.com/jenskutilek/homecomputer-fonts at commit https://github.com/jenskutilek/homecomputer-fonts/commit/9b182d8ab290d98477ddc0a4f84c8137d540994b.

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Updated

## Workbench: Version 2.000 added

---
 f45a382: [gftools-packager] Workbench: Version 2.000 added

* Workbench Version 2.000 taken from the upstream repo https://github.com/jenskutilek/homecomputer-fonts at commit https://github.com/jenskutilek/homecomputer-fonts/commit/9a28003a6419e7ebd9d81469288a80e436866823.

**Comment by vv-monsalve** [comment]
- 🔥 FAIL: Does DESCRIPTION file contain broken links? 

The link will exist when #6919 is merged. We should remember to counter-check the link once in prod.

- 🔥 FAIL: Check Google Fonts glyph coverage. 
- 🔥 FAIL: Check font follows the Google Fonts vertical metric schema 

Due to the historical origin of the fonts, it was agreed to accept these fails.

- 🔥 FAIL: Check variable font instances

Given the font only includes custom axes, it is accepted for them to include custom-named instances. After Kablammo's case, this was confirmed for the homecomputer fonts. I need to include this in the GF-Guide

- 🔥 FAIL: Check a font's STAT table contains compulsory Axis Values.

Bug reported in https://github.com/googlefonts/axisregistry/issues/99#issuecomment-1770941559

- 🔥 FAIL: Validate METADATA.pb axes tags are defined in gf_axisregistry.

The SCAN and BLED axes were already merged in the googlefonts/axisregistry repo and will be brought with an upcoming git subtree pull. **Please do not merge this PR until I create that pull.**

- ⚠ WARN: METADATA.pb: Designers are listed correctly on the Google Fonts catalog?

Jens Kutílek bio info was added in #6935

File: ofl/workbench/DESCRIPTION.en_us.html
Comment (emmamarichal) [inline ofl/workbench/DESCRIPTION.en_us.html:1]: Same as Sixtyfour, with the "I" instead of "he" :)

Comment (vv-monsalve) [reply ofl/workbench/DESCRIPTION.en_us.html:1]: Done

**Comment by RosaWagner** [comment]
@vv-monsalve the "regular" instance correspond to the "close" location but is called "regular" in the FVAR table 
Screenshot 2023-11-15 at 14 31 15

**Comment by RosaWagner** [comment]
After discussing the axes with @vv-monsalve, we decided to merge the PR so see how they render in sandbox but she would make a new PR where "Close" is elided in the STAT table for better cross-platform naming consistency.

**Review by RosaWagner — APPROVED** [review]

**Pull request merged**